MAINTAINERS.md

```diff
@@ -56,15 +158,30 @@ packages (to enable `agoric install <TAG>`), use:
 ./scripts/npm-dist-tag.sh lerna add <TAG>
 ```
 
-To make validators' lives easier, create a Git tag for the chain-id:
+- [ ] Push release labels as tags
+
+Perform the following for each `tag` that we will use to label this release.
 
 ```sh
-CHAIN_ID=agoricstage-27 # Change this as necessary
-git tag -s -m "release $CHAIN_ID" $CHAIN_ID @agoric/sdk@$SDKVER^{}
-git push origin $CHAIN_ID
-./scripts/docker-tag.sh $SDKVER $CHAIN_ID
+git tag $tag
+git push origin $tag
 ```
 
+- [ ] Confirm that a Docker image for SDKVER has been published.
+
+- [ ] Create a GitHub release
+
+Follow the [GitHub
+instructions](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository
+) and use previous releases as a template. This uses the
+validator-oriented release description.
+
+### Cleanup
+
+- [ ] Review recent changes in the base branch for anything that
+should be merged into its ancestors, all the way up to master. This
+should include the changes to CHANGELOG.md.
+
```

Comment:
maybe also recommend deleting the `agoric-sdk` checkout that you used, when you're all done, to avoid confusion when you make the next release (don't re-use the temporary tree)

Reply:
Done.